tests/test_connection.py

```diff
@@ -172,7 +172,49 @@ def test_lists_connection_information(runner):
                 "database": "dev_database",
                 "host": "dev_host",
                 "port": 8000,
-                "protocol": "dev_protocol",
```

Comment:
I had to remove `protocol` from expected output because it's not supported via `connection add`, so we won't print it in `connection list`